# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/nvi-syssiglist.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./ex/filter.c.orig	2005-01-19 00:27:53.000000000 -0300
+++ ./ex/filter.c	2005-01-19 00:28:48.000000000 -0300
@@ -372,7 +371,7 @@
 		len = strlen(p);
 		msgq(sp, M_ERR, "%.*s%s: received signal: %s%s",
 		    MIN(len, 20), p, len > 20 ? " ..." : "",
-		    sys_siglist[WTERMSIG(pstat)],
+		    strsignal(WTERMSIG(pstat)),
 		    WCOREDUMP(pstat) ? "; core dumped" : "");
 		if (nf)
 			FREE_SPACE(sp, p, 0);
